stats/lib/platform-routes.js

```diff
@@ -36,6 +38,8 @@ export const handlePlatformRoutes = async (req, res, pgPools) => {
     await respond(pgPools.evaluate, fetchParticipantsWithTopMeasurements)
   } else if (req.method === 'GET' && url === '/participants/top-earning') {
     await respond(pgPools.stats, fetchTopEarningParticipants)
+  } else if (req.method === 'GET' && url === '/participants/summary') {
+    json(res, await fetchParticipantsSummary(pgPools.evaluate))
```

Comment:
Hang on. The SQL query takes more than 400ms. I am not comfortable running that query on every request; we need to cache the result.

Reply:
410b9cc